doc/source/install.rst

```diff
@@ -60,14 +64,6 @@ Then:
     $ python setup.py install
```

Comment:
I think today the practice is `pip install .` no ?

Reply:
I don't really know; I've never encountered a use case for manually downloading and installing the package this way, I just left it in as is.